understanding/20/audio-description-prerecorded.html

```diff
@@ -16,12 +16,22 @@ <h2>Intent of Audio Description (Prerecorded)</h2>
       <p>The intent of this Success Criterion is to provide people who are blind or visually
          impaired access to the visual information in a synchronized media presentation. The
          audio description augments the audio portion of the presentation with the information
-         needed when the video portion is not available. During existing pauses in dialogue,
-         audio description provides information about actions, characters, scene changes, and
-         on-screen text that are important and are not described or spoken in the main sound
-         track.
+         needed when the video portion is not available. Audio description provides information about actions,
+         characters, scene changes, and on-screen text that are important and are not described
+         or spoken in the main sound track. Audio description is usually added during existing
+         pauses in dialogue. In rare cases where the original audio does not provide sufficient
+         pauses, it may be possible to create them by lowering the volume ("ducking") some of the
+         existing dialogue, if the need for audio description is deemed to be more important.
       </p>
       
+      <div class="note">
+         <p>This success criterion applies even in cases where the original audio does not
```

Comment:
If the group decides to indeed absolve/give a pass to videos/multimedia presentations that have no gaps https://github.com/w3c/wcag/issues/1768#issuecomment-839360858 then this note should probably be changed to

```
This success criterion does not apply in cases where the original audio does not have sufficient pauses, and where no portion of the original audio can be omitted in favour of an audio description without losing important information. In these cases, the content will not fail this success criterion, as it would either require the content to be redone with sufficient pauses for audio description, or the use of extended audio description, which is covered at level AAA by <a href="#extended-audio-description-prerecorded">1.2.7 Extended Audio Description (Prerecorded)</a>
```

Reply:
taking @detlevhfischer's suggestion to the above comment (noting it's not part of any PR, just a side note here) about removing "redone", something like

```
<p>This success criterion does not apply in cases where the original audio does not
have sufficient pauses, and where no portion of the original audio can be omitted in
favour of an audio description without losing important information. In these cases,
the content will not fail this success criterion, as it would either require the content to
be edited to have sufficient pauses for audio description, or the use of extended audio
description, which is covered at level AAA by 
<a href="#extended-audio-description-prerecorded">1.2.7 Extended Audio Description (Prerecorded)</a>.</p>
```